ratpack-kafka-producer/src/test/groovy/smartthings/ratpack/kafka/KafkaProducerServiceSpec.groovy

```diff
@@ -10,14 +10,18 @@ import spock.lang.Specification
 
 class KafkaProducerServiceSpec extends Specification {
 
+	String getTestKafkaServers() {
+		return System.getenv("KAFKA_SERVER") ?: '192.168.99.100:9092'
```

Comment:
Would it make sense to not have a default here at all and blow up if it isn't set?